packages/eslint-plugin-shopify/lib/config/rules/stylistic-issues.js

```diff
@@ -88,7 +88,9 @@ module.exports = {
   // Disallow whitespace before properties
   'no-whitespace-before-property': 'error',
   // Require or disallow padding inside curly braces
-  'object-curly-spacing': 'off',
+  'object-curly-spacing': ['warn', 'never'],
```

Comment:
I was just bringing this in line with our actual recommendation, good catch that it doesn't match the Ruby recommendation. I think there is reason enough to have this difference — since JS blocks are denoted with braces, it is helpful to have a different brace style between objects and functions to differentiate them. Ruby doesn't have this problem since any multiline block is usually not brace-delimited, and manipulating hashes is probably less common than in JS. Also, JS's destructuring, especially in parameters, means that the extra space inside of braces ends up making things quite a bit harder to read.